### DISCUSSION TITLE
FLUX doesn't understand specifics!

### DISCUSSION THREAD

**Discussion by bsdesigner** [lllyasviel/stable-diffusion-webui-forge#1494] [General]
I came across the fact that Flux doesn't understand many things that SDXL models understood quite well. For example, Flux doesn't understand when I ask it to generate a celebrity, be it a famous actor, musician, politician or something else. Of course it tries, but it doesn't look like them at all. 

Also Flux doesn't understand the styles of famous artists, photographers, film directors or architects. If I ask SDXL model to generate me an image in the style of: Mike Deodato or Rembrandt, Fred Herzog or Greg Girard, Aki Kaurismäki or Christopher Nolan, Tadao Ando or Zaha Hadid, SDXL model will cope with it and will generate me an image with maximum hit in the style. But Flux model, generates without absolute understanding of the name of the one whose style should be orientated on....

Maybe I have not downloaded and installed something? Maybe I'm not setting it up right? Is this just my problem or is someone else also facing this problem?...?!

**Comment by abzaloff** [comment]
Flux is a fairly new model and does not know a lot of things yet. And Forge has nothing to do with it. You need to ask the authors of the model a question, not in the Forge topic

**Reply by bsdesigner** [reply]
Did I say anything about Forge? I have no complaints about Forge, it's a great product. I asked for advice from the community about a particular model and nothing more. And, where exactly to ask questions I will decide without your advice...

**Reply by abzaloff** [reply]
I tell you again that Forge is being discussed here, not the capabilities of a particular model. If Flux doesn't understand you, Forge has nothing to do with it. Calm down and look for answers where they are.

**Reply by ScionoicS** [reply]
> Did I say anything about Forge?  

/stable-diffusion-webui-forge/discussions/

Take note of the URL. This isn't reddit. This is a specific discussion group.

**Comment by Pdonor** [comment]
I think that for safety/pr reasons, a lot of artist and female celeb names were removed from the labels of the training data. But this seems different from previous 'censored' models, because the missing concepts and people pop right back in with a LORA. I recall it took months to get genitals into SD 1.5 and SDXL, and it only took days for Flux.

**Reply by bsdesigner** [reply]
To hell with him if he doesn't understand the appearance of celebrities. But he doesn't understand the styles in art and the styles in which the most famous artists, photographers, architects, designers and film-makers worked! And this is nonsense! 

Here's an example: I'm in the process of making a video about San Francisco in the late 1960s, and I need images in the style of street photographers Fred Herzog and Philip-Lorca diCorcia. Now, Flux can't understand that era, the fashion of that period, or the style in which these photographers worked. SDXl understands, but Flux doesn't..... I've already written long, long prompts describing all the details and particulars, but there is no effect. Images he makes very high quality, but with understanding the specifics he has trouble....

**Comment by vonespy** [comment]
Let me see if this helps. You buy a house from builder X. The house has furnace installed by Y furnace. When you have an issue with your furnace, you don't call your builder, you call Y furnace. In this case, forge is the house, flux is the furnace. Posting problems on a forge forum is just going to irritate a lot of people without accomplishing anything useful.

Flux is a MLM used by forge. What people are trying to tell you is that you are complaining about a **flux model from [black forest labs](https://github.com/black-forest-labs/flux)** issue in the **forge** forum. 

Your best bet will be talking to [black forest labs](https://github.com/black-forest-labs/flux) about what you feel is missing from flux.

Edit: guys, OP is from Ukraine so I suspect this might just be some ideas getting lost in translation. No shame, I'm a 1 language 'merican so anyone that knows more languages than me is a god in my eyes.

**Reply by bsdesigner** [reply]
Yes, I'm from Ukraine and I'm a former military man who spent a year in the infantry, but now I've been discharged due to injury and I'm working on artificial intelligence. And yes, I don't know English well enough and that's why I use Deepl.

Now to the point. The point of my question was that maybe I didn't download and install something in Forge? Maybe the problem is not in the model, but in the fact that I have not installed something that would help the model to understand my prompt? This was actually the question I asked, but not very attentive commenters, not having read the question to the end and not understanding the essence of the question began to write some useless comments, and even with some claims. Since I wrote not only in the Forge forum, but also in other forums, it is now obvious to me that the issue is not with me or the fact that I have not installed something, but with the Flux model itself. That's it! For me personally, the question has been removed. But maybe someone else will have such a question and after reading this thread will be able to find answers for themselves....

Thank you all! Good luck and clear skies!

**Reply by vonespy** [reply]
No shame, my friend. So much gets lost in translation between languages. 

But you're right. I think the issue here is that the flux model just wasn't trained in some of the areas that we'd like it to be trained. I'm finding a lot of popular culture that flux just doesn't know about, but I'm guessing that black forest labs would be eager for feedback on how they can improve their model.

You may also be running into areas of training that aren't happening because model training can be very United States centric. Again, I suspect that black forest labs would be interested in your Ukraine feedback. 

I'm certainly not an expert at any of this stuff but would be happy to help a former Ukrainian infantryman if I can. Let me know if you'd like to take this conversation to reddit or facebook or bluesky.

**Reply by bsdesigner** [reply]
To my shame, I have to admit that this is the first time I've heard of ‘Bluesky’ )) Need to sign up!
